.github/workflows/release.yaml

```
@@ -174,6 +174,8 @@ jobs:
 
   verification-test:
     runs-on: ${{ matrix.os }}
+    env:
+      HOLOSCAN_VERSION: 3.0.0
```

Should this use the latest version?
```suggestion
    env:
      HOLOSCAN_VERSION: 3.3.0
```

---

No, any version would do here as we want to ensure the output is correct.